app/components/ui/Footer.jsx

```diff
@@ -5,14 +5,40 @@ function Footer() {
   return (
     <footer className="site-footer" role="contentinfo">
       <div className="footer-content">
+        <div className="footer-top">
+          <section className="footer-service-guide">
+            SF SERVICE GUIDE
+          </section>
+          <section className="footer-links">
+            <ul>
+              <li>ABOUT US</li>
+              <li><a href="sheltertech.org">Our Story</a></li>
+              <li><a href="sheltertech.org/volunteer">Volunteer</a></li>
+              <li><a href="sheltertech.org/donate">Donate</a></li>
+            </ul>
+            <ul>
+              <li>HELP</li>
+              <li><a href="#">FAQ</a></li>
+              <li><a href="#">Digital Literacy</a></li>
```

Comment:
Same for these, though @derekfidler I guess the specs for this links is a direct request from the city? We need content for them otherwise we should comment the links out until we can add the pages.

Reply:
@richardxia @lexholden I will comment out these links until we know where they go. That way the commit can be unblocked.